app/src/main/java/org/sopt/pingle/presentation/ui/main/plan/PlanViewModel.kt

```diff
@@ -45,10 +47,28 @@ class PlanViewModel : ViewModel() {
     private val _address = MutableStateFlow<String?>(null)
     val address get() = _address.asStateFlow()
 
-    // TODO 뷰 연결 시 버튼 활성/비활성화 로직 isPlanBtnEnabled에 추가
     private val _selectedCategory = MutableStateFlow<CategoryType?>(null)
     val selectedCategory get() = _selectedCategory.asStateFlow()
 
+    private val _selectedRecruitment = MutableStateFlow<String?>("1")
+    val selectedRecruitment get() = _selectedRecruitment.asStateFlow()
+
+    fun incRecruitmentNum() {
+        var i = selectedRecruitment.value?.toInt()
+        i = i!! + 1
+        setSelectedRecruitment(i.toString())
+    }
+
+    fun decRecruitmentNum() {
+        var i = selectedRecruitment.value?.toInt()
+        i = i!! - 1
+        setSelectedRecruitment(i.toString())
+    }
+
+    fun setSelectedRecruitment(recruitment: String) {
+        _selectedRecruitment.value = recruitment
+    }
+
```

Comment:
```suggestion
    fun setSelectedRecruitment(recruitment: Int) {
        _selectedRecruitment.value = recruitment.toString()
    }

```

Int 값을 대입해주는 경우가 많기 때문에 함수 안에서 스트링 변환을 수행해주어도 좋을 것 같습니당